### ISSUE TITLE
disable help lines when clicking

### ISSUE THREAD

**Issue by GoogleCodeExporter** [thaindq/opensudoku-android#33]
```
Hello,

I love sudoku and I usually don't play with hints, the actually distract me. 
A wonderful feature would be to disable the extra lines that are drawn now.

I do like the delay that happens when you press a field, that really helps.

--
Henk van der Velden

```

Original issue reported on code.google.com by `henkvand...@gmail.com` on 12 Sep 2009 at 9:18

**Comment by GoogleCodeExporter** [comment]
```
Issue 28 has been merged into this issue.
```

Original comment by `romario...@gmail.com` on 16 Sep 2009 at 2:21

**Comment by GoogleCodeExporter** [comment]
```
Issue 29 has been merged into this issue.
```

Original comment by `romario...@gmail.com` on 16 Sep 2009 at 2:21

**Comment by GoogleCodeExporter** [comment]
```
Issue 30 has been merged into this issue.
```

Original comment by `romario...@gmail.com` on 16 Sep 2009 at 2:23

**Comment by GoogleCodeExporter** [comment]
```
Issue 31 has been merged into this issue.
```

Original comment by `romario...@gmail.com` on 16 Sep 2009 at 2:23

**Comment by GoogleCodeExporter** [comment]
```
Issue 32 has been merged into this issue.
```

Original comment by `romario...@gmail.com` on 16 Sep 2009 at 2:24

**Comment by GoogleCodeExporter** [comment]
```
I've added option "Highlight cell on touch" (in Game Settings/Game/More 
settings).

Delay that happens when you press a field is not intentional, it's just that 
slow ;-).

Issue is solved in 0.7.1, if you want to test it before official release, you 
can
download it here: http://code.google.com/p/opensudoku-android/downloads/list 
(look
for OpenSudoku 0.7.1 - unstable). Please note that if you install this test 
version,
you will probably no longer receive updates from market, so when you're 
finished with
testing, install OpenSudoku from market again.
```

Original comment by `romario...@gmail.com` on 29 Sep 2009 at 2:46
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `romario...@gmail.com` on 2 Oct 2009 at 5:45
- Changed state: **Fixed**
- Added labels: **Milestone-0.7.1**
